config/jobs/kubernetes/sig-testing/bazel-build-test.yaml

```diff
@@ -109,7 +109,7 @@ postsubmits:
               --config=ci \
               "--build=//... -//vendor/..." \
               --release=//build/release-tars \
-              --gcs=gs://kubernetes-release-dev/ci \
+              --gcs=gs://kubernetes-release-dev/ci-rbe \
```

Comment:
this one definitely seems reasonable, this job should probably be de-duped to go one way or the other @fejta